lib/game.rb

```diff
@@ -6,18 +6,24 @@ class Game
   def initialize(input:, output:)
     @input = input
     @output = output
+    @grid = {}
   end
 
   def play
     grid_view.render
     get_player_input
+    grid_view.render
   end
 
   private
 
-  attr_reader :input, :output
+  attr_reader :grid, :input, :output
 
   def get_player_input
+    grid[player_coordinates] = "X"
```

Comment:
This line seems very funky and points to the data-ness of the `grid`. It gets a bit weirder when that same data-ness is present in the grid view. 

I would say that everything that needs to interact with the grid of data would get safer if there were a `Grid` class.